cdap-ui/app/cdap/components/AbstractWidget/MultiStageSelector/index.tsx

```diff
@@ -36,6 +37,7 @@ const styles = () => {
 
 interface IMultiStageSelectorWidgetProps {
   delimiter: string;
+  singleSelectOnly: boolean;
```

Comment:
Should this be optional?

Reply:
I think so, but I was following the convention for `delimiter`. I think thats technically an optional parameter (at least it is for the underlying MultiSelect) but we didnt mark it as optional here. Wasnt sure if that was important somehow. I can make it optional.